### DISCUSSION TITLE
Window does not open by <C-f> for filtering language. Opening only full screen numbered language list.

### DISCUSSION THREAD

**Discussion by nPaul** [mason-org/mason.nvim#1460] [Q&A]
Window does not open by <C-f> for filtering language. Opening only full screen numbered language list

NVIM v0.9.1 compiled from source.

**Comment by williamboman** [comment]
Mason uses `vim.ui.select` to prompt for a selection. The default one is what you described, the screenshot in the README uses https://github.com/stevearc/dressing.nvim.
